rules/no-ignored-test-files.js

```diff
@@ -8,7 +8,9 @@ var createAvaRule = require('../create-ava-rule');
 var defaultFiles = [
 	'test.js',
 	'test-*.js',
-	'test/**/*.js'
+	'test/**/*.js',
+	'**/__tests__',
```

Comment:
Should be `**/__tests__/**/*.js` as this plugin doesn't have directory recursion like AVA.